src/main/java/ladder/domain/Point.java

```diff
@@ -1,62 +1,43 @@
 package ladder.domain;
 
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
+import java.util.Objects;
 
 public class Point {
-    public static final String EXCEPTION_BOTH_TRUE = "가로 라인은 한 좌표에 하나만 가질 수 있습니다.";
-    private static final boolean INITIAL_LEFT = false;
-    private static final boolean INITIAL_RIGHT = false;
-    private boolean left;
-    private boolean right;
-
-    public Point() {
-        this(INITIAL_LEFT, INITIAL_RIGHT);
-    }
+    private final Direction direction;
 
     public Point(boolean left, boolean right) {
-        if (left && right) {
-            throw new IllegalArgumentException(EXCEPTION_BOTH_TRUE);
-        }
-        this.left = left;
-        this.right = right;
+        this.direction = Direction.of(left, right);
     }
 
-    public static List<Point> createInitialPoints(int height) {
-        return IntStream.range(0, height)
-                .mapToObj(i -> new Point())
-                .collect(Collectors.toList());
+    public static Point first(boolean right) {
+        return new Point(false, right);
     }
 
-    public boolean isLeft() {
-        return left;
+    public Point last() {
+        return new Point(direction.isRight(), false);
     }
 
-    public boolean isRight() {
-        return right;
-    }
-
-    private void connectLeft() {
-        if (isRight()) {
-            throw new IllegalArgumentException(EXCEPTION_BOTH_TRUE);
+    public Point next(boolean nextRight) {
```

Comment:
전략 패턴을 사용하고 있으니 포인트를 생성할 때 전략을 주입받으면 어떨까요?
```suggestion
    public Point next(DrawStrategy drawStrategy) {
```